src/erlfmt.erl

```diff
@@ -144,11 +144,11 @@ insert_pragma_node(Node) ->
         case PreComments of
             [] ->
                 Loc = erlfmt_scan:get_anno(location, Node),
-                [{comment, #{location => Loc, end_location => Loc}, ["%% @format", ""]}];
+                [{comment, #{location => Loc, end_location => Loc}, ["%%% % @format", ""]}];
             _ ->
                 {comment, Loc, LastComments} = lists:last(PreComments),
                 lists:droplast(PreComments) ++
-                    [{comment, Loc, LastComments ++ ["%% @format"]}]
+                    [{comment, Loc, LastComments ++ ["%%% % @format"]}]
```

Comment:
I feel like the proper way to do this would be to detect whether to use `%%` or `%%%`, In particular something like:

```erl
{Prefix, _} = string:take(lists:last(LastComment), "%"),
[{comment, Loc, LastComments ++ [Prefix ++ " % @format"]}]
```

I don't have a preference for what should be the default in case there weren't any comments already.

Reply:
We could, but we still need a default for the case where there are no comments yet and in this case I think we should do 3 comments, given this new analyses: https://github.com/WhatsApp/erlfmt/pull/283

Reply:
I have added your suggestion :)